cmd/iam_apikey_list.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type apiKeyItem struct {
+	Description string              `json:"description"`
+	Key         string              `json:"key"`
+	Operations  string              `json:"operations,omitempty"`
+	Type        egoscale.APIKeyType `json:"type"`
+}
+
+type apiKeyListItemOutput []apiKeyItem
+
+func (o *apiKeyListItemOutput) toJSON()  { outputJSON(o) }
+func (o *apiKeyListItemOutput) toText()  { outputText(o) }
+func (o *apiKeyListItemOutput) toTable() { outputTable(o) }
+
+// apiKeyListCmd represents the List command
+var apiKeyListCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List APIKeys",
+	Aliases: gListAlias,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
```

Comment:
To be consistent with the other functions we don't put a space at the beginning of the function